pipeline-pullrequests.yaml

```diff
@@ -139,6 +139,31 @@ jobs:
           path: pull-request
           status: failure
 
+    - task: build-kyt-api-test-server
+      image: image-golang
+      config:
+        platform: linux
+        inputs:
+          - name: pull-request
+          - name: openapi-api-server
+        run:
+          path: /bin/bash
+          args:
+            - -ec
+            - |
+              ROOT=$(pwd)
+              mkdir -p ${ROOT}/pull-request/kyt-api-server/openapi
+              cp -r openapi-api-server/openapi/model_device.go ${ROOT}/pull-request/kyt-api-server/openapi
+              cp -r openapi-api-server/openapi/routers.go ${ROOT}/pull-request/kyt-api-server/openapi
```

Comment:
Use those two lines instead:
```
rm openapi-api-server/openapi/api_device.go
cp -r openapi-api-server/openapi/* ${ROOT}/pull-request/kyt-api-server/openapi
```
This change was introduced in #14 and is a more generic way of doing things.